src/containers/AccessPointDetails/components/OS/components/Timer/index.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import moment from 'moment';
+
+const Timer = () => {
```

Comment:
this should be combined with the circle countdown, and this doesn't need to count up. Just set the date on refresh. See my comments below for what you did wrong anyways.